source/docs/software/advanced-controls/introduction/control-system-basics.rst

```diff
@@ -33,6 +44,15 @@ The figure below shows a system with a hypothetical input and output. Since the
 .. image:: images/control-system-basics-whatisgain.png
    :alt: A system diagram with hypothetical input and output
 
+What is a Model?
+----------------
+
+A *model* of your mechanism is a set of math equations which describe its expected behavior.
+
+They can be built up from first-principles physics equations, and refined by doing experiments on the mechanism.
+
+Models are often simple algebra equations. They can also include some linear algebra, matricies, and differential equations. WPILib provides a number of classes to help simplify the more complex math.
```

Comment:
These three sentences could use some transitional vocabulary. They are sort of "shoved" together if that makes any sense.